ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+If making bug report please copy and paste in to the issue the output of 

```

Comment:
Grammar:
If you are making a bug report, please copy and paste the output of the following Julia script into the issue: